### ISSUE TITLE
Support brims are ignored by automatic arrange function

### ISSUE THREAD

**Issue by Lecso11** [supermerill/SuperSlicer#2663]
### What happened?

When placing supports, and then using automatic arrange, the support brim hits the model, and are overlapped.
![kép](https://user-images.githubusercontent.com/78912917/162377677-31bf1eb2-1a27-4c81-b688-ec61de131564.png)

### Project file & How to reproduce

[satorvas.zip](https://github.com/supermerill/SuperSlicer/files/8449169/satorvas.zip)

### Version

2.4.58.2

### Operating system

Windows 10

### Printer model

Voron 2.4 with klipper

**Comment by neophyl** [comment]
Correct.  Its that way in PS too, always has been.  You can set the arrange distance to be greater in this case or you can merge the parts into a single object and then the support/brims etc treat it as a single object and don't over lap.

Its a side effect of the ability to slice different objects with different layer heights etc.  
There are multiple issues open on the PS github for it, for example https://github.com/prusa3d/PrusaSlicer/issues/3324 and also ones here too such as this recent one https://github.com/supermerill/SuperSlicer/issues/2606 but at the moment this is expected operation.
